.github/workflows/build-pr.yml

```diff
@@ -50,30 +53,34 @@ jobs:
 
   build_test_e2e:
     name: "Build and run e2e Test"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     timeout-minutes: 15
     permissions:
       contents: read
+      id-token: write
     strategy:
       matrix:
         KUBERNETES_VERSION: ["1.24.6"]
```

Comment:
Any plans of expanding this matrix to test against all supported Kubernetes releases? (xref: https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/.github/workflows/e2e.yaml#L27).

Reply:
it's mentioned in the test issue: https://github.com/deislabs/ratify/issues/324, I can add a few versions to see if Ratify works.